dune/stuff/common/compiler.hh

```diff
@@ -7,14 +7,11 @@
 #define DUNE_STUFF_COMMON_COMPILER_HH
 
 #if defined(__GNUC__)
-# if defined(__GNUC_PATCHLEVEL__)
-#  define DS_GNUC_VERSION (__GNUC__ * 10000 \
-                            + __GNUC_MINOR__ * 100 \
-                            + __GNUC_PATCHLEVEL__)
-# else
-#  define DS_GNUC_VERSION (__GNUC__ * 10000 \
-                            + __GNUC_MINOR__ * 100)
-# endif
+#if defined(__GNUC_PATCHLEVEL__)
+#define DS_GNUC_VERSION (__GNUC__ * 10000 + __GNUC_MINOR__ * 100 + __GNUC_PATCHLEVEL__)
+#else
+#define DS_GNUC_VERSION (__GNUC__ * 10000 + __GNUC_MINOR__ * 100)
+#endif
 #endif
```

Comment:
Yes, I saw those (and already applied them in 132f83eb3696aa3412a564e5b49846c9d11356af, since I had the feeling that clang-format would actually break code there, but that was just a guess). I will see if clang-format respects those even if they are trailing comments...